main/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+
+class User(models.Model):
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=50)
+    nickname = models.CharField(max_length=30, unique=True)
```

Comment:
It is usually called `username`
```suggestion
    username = models.CharField(max_length=30, unique=True)
```

Reply:
Username exists in Django user model, this line was remowed